src/helpers/productsBought.js

```diff
@@ -0,0 +1,12 @@
+import {inventory} from "../constants/inventory.js";
```

Comment:
Je wilt bij een helper functie geen imports hebben die betrekking hebben op wat er in de functie gebeurt. Op deze manier is de helper functie nu alleen bruikbaar voor die specifieke array/object. Je wilt dat een helper functie herbruikbaar is. Dit kan je het beste doen door gebruik te maken van parameters/argumenten.